lib/modules/platform/gitea/index.ts

```diff
@@ -81,6 +82,11 @@ let config: GiteaRepoConfig = {} as any;
 let botUserID: number;
 let botUserName: string;
 
+function isReconfigurePr(branchName: string): boolean {
+  logger.debug({ branchName }, 'isReconfigurePr');
+  return regEx(/reconfigure$/g).test(branchName);
```

Comment:
do we really need this logging for every PR?

I think we should just inline the regex test and store the regex outside the loop for performance